database-seed/remote-image-getter.js

```diff
@@ -3,52 +3,59 @@ const path = require('path');
 const axios = require('axios');
```

Comment:
not a huge deal, but I don't think this file needs to be in your repo at all since it's just a way for you to download a bunch of images and within the repo you're just using the s3 link for each image. Definitely cool that you wrote your own script for this though